### PR TITLE
Update README: add sections about required OSE config and role vars description

### PR DESCRIPTION
**Pull request by vrutkovs**
PTAL @twaugh @lcarva

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,83 @@ Playbook for deploying OSBS into [OpenShift Dedicated](https://www.openshift.com
...
+
+Specify project namespace
+```
+osbs_namespace: best-namespace
+```
+
+Create service accounts. 
+
+Depending on your configuration OSBS needs 
+service accounts to communicate with koji and pulp (see osbs-client configuration).
```
Comment (twaugh) [inline README.md]: Aren't service accounts more about allowing e.g. koji to communicate with OSBS? It's just an account that a service (e.g. koji) can use to authenticate itself with OpenShift.

I don't think Pulp has a service account, as it does not need to make any requests to OpenShift.

File: README.md [hunk 2]
```diff
@@ -12,3 +12,83 @@ Playbook for deploying OSBS into [OpenShift Dedicated](https://www.openshift.com
...
+osbs_readwrite_users:
+  - system:serviceaccount:{{ osbs_namespace }}:koji
+  - system:serviceaccount:{{ osbs_namespace }}:builder
+osbs_readwrite_groups:
+  - system:authenticated
+osbs_admin_users: []
+osbs_admin_groups: []
+```
+
+Set maximum number of simultanious builds
```
Comment (twaugh) [inline README.md]: typo

File: README.md [hunk 3]
```diff
@@ -12,3 +12,83 @@ Playbook for deploying OSBS into [OpenShift Dedicated](https://www.openshift.com
...
+```
+osbs_master_max_pods: -1
+```
+
+Skip errors on importing secrets, if those are not required
+```
+osbs_secret_can_fail: true
+```
+
+Read pulp credentials from secret and put them in a local dir
```
Comment (twaugh) [inline README.md]: It's the other way around I think: the credentials are read from the local directory and used to create the secret.

File: README.md [hunk 4]
```diff
@@ -12,3 +12,83 @@ Playbook for deploying OSBS into [OpenShift Dedicated](https://www.openshift.com
...
+koji_secret_name: kojisecret
+koji_secret_local_dir: /home/user/directory-with-koji-certificates
+```
+
+Hardcoded vars
+==============
+
+The following vars are used in other roles:
+
+Set enviroinment variables (e.g. http_proxy) to be used for some commands
```
Comment (twaugh) [inline README.md]: typo

File: README.md (hunk 1 above)
Comment (vrutkovs) [reply README.md]: Correct, rewrote this part

File: README.md (hunk 2 above)
Comment (vrutkovs) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (vrutkovs) [reply README.md]: Right, updated this

File: README.md (hunk 4 above)
Comment (vrutkovs) [reply README.md]: Fixed

File: README.md [hunk 5]
```diff
@@ -12,3 +12,83 @@ Playbook for deploying OSBS into [OpenShift Dedicated](https://www.openshift.com
...
+
+Role Variables
+==============
+
+Specify project namespace
+```
+osbs_namespace: best-namespace
+```
+
+Create service accounts.
```
Comment (twaugh) [inline README.md]: nitpick: doesn't need a full-stop here if it's a sort of heading

**Comment by twaugh** [comment]
LGTM

**Comment by lcarva** [comment]
LGTM

**Pull request merged**